klaam/__init__.py

```diff
@@ -0,0 +1 @@
+from klaam.run import *
```

Comment:
I really disagree with the wild card imports as it results in a lot of name space conflicts. I think `from klaam.run import SpeechRecognition SpeechClassification TextToSpeech` should nail it?

Reply:
I agree with you. I will update this to be more explicit to select modules.